### PR TITLE
LPS-99983 Use new property name for workspace

### PR DESCRIPTION
**Pull request by christopherbryanboyd**

### CODE REVIEW

**Comment by christopherbryanboyd** [comment]
ci:test:relevant

**Comment by christopherbryanboyd** [comment]
ci:test:gradle-plugins

**Comment by christopherbryanboyd** [comment]
ci:test:sf

**Comment by christopherbryanboyd** [comment]
ci:test:bundle

**Comment by christopherbryanboyd** [comment]
ci:test:sf

**Comment by christopherbryanboyd** [comment]
ci:test:sf

**Comment by christopherbryanboyd** [comment]
ci:test:sf

**Comment by christopherbryanboyd** [comment]
ci:test:gradle-plugins

**Comment by christopherbryanboyd** [comment]
ci:test:relevant

**Comment by christopherbryanboyd** [comment]
ci:test:bundle

File: modules/sdk/gradle-plugins-workspace/src/gradleTest/preCompileJSPDisabled/build.gradle [hunk 1]
```diff
@@ -0,0 +1,22 @@
+task runGradleTest
+
+gradle.liferayWorkspace {
+	homeDir = new File(buildDir, "bundles")
+	modules.jspPrecompileEnabled = false
+}
+
+runGradleTest {
+	dependsOn ":modules:foo-web:assemble"
+	dependsOn ":modules:foo-web2:assemble"
```
Comment (gamerson) [inline modules/sdk/gradle-plugins-workspace/src/gradleTest/preCompileJSPDisabled/build.gradle]: where is `foo-web2`?

File: modules/sdk/gradle-plugins-workspace/src/gradleTest/preCompileJSPDisabled/build.gradle [hunk 2]
```diff
@@ -0,0 +1,22 @@
+task runGradleTest
+
+gradle.liferayWorkspace {
+	homeDir = new File(buildDir, "bundles")
+	modules.jspPrecompileEnabled = false
+}
+
+runGradleTest {
+	dependsOn ":modules:foo-web:assemble"
```
Comment (gamerson) [inline modules/sdk/gradle-plugins-workspace/src/gradleTest/preCompileJSPDisabled/build.gradle]: also you can put two things on the `dependsOn` line, like
```
dependsOn ":modules:foo-web", ":modules:bar-web:assemble"
```
Or better yet you could do
```dependsOn "assemble"```

**Comment by christopherbryanboyd** [comment]
@gamerson  I have updated the PR with the requested changes.

**Comment by christopherbryanboyd** [comment]
ci:test:sf

**Comment by christopherbryanboyd** [comment]
ci:test:gradle-plugins

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by christopherbryanboyd** [comment]
ci:test:gradle-plugins

**Comment by christopherbryanboyd** [comment]
ci:test:sf

**Comment by christopherbryanboyd** [comment]
ci:test:relevant

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/gamerson/liferay-portal/pull/341. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**